src/crates-and-source-files.md

```diff
@@ -20,9 +20,8 @@
 Rust's semantics obey a *phase distinction* between compile-time and
 run-time.[^phase-distinction] Semantic rules that have a *static
 interpretation* govern the success or failure of compilation, while
-semantic rules
-that have a *dynamic interpretation* govern the behavior of the program at
-run-time.
+semantic rules that have a *dynamic interpretation* govern the behavior of the
+program at run-time.
```

Comment:
Unrelated, but does this distinction ever appear again?

Reply:
Const functions perhaps? It's a paragraph that's survived since the reference was originally the guide.